examples/platform/silabs/efr32/wf200/efr_spi.c

```diff
@@ -316,55 +287,24 @@ sl_status_t sl_wfx_host_spi_transfer_no_cs_assert(sl_wfx_host_bus_transfer_type_
     return SL_STATUS_OK;
 }
 
-/****************************************************************************
```

Comment:
Now why removed function headers ? Don't we want the documentation ?

Reply:
the documentation has been moved to the common header in `spi_multiplex.h`